### PR TITLE
Run unit tests in circle and include their coverage in the report.  

### PR DESCRIPTION
**Pull request by tomwilkie**
Report coveralls.io from Circle.

### CODE REVIEW

File: test/units.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+go get github.com/weaveworks/weave/testing/cover
```
Comment (rade) [inline test/units.sh]: It seems very strange to refer to the weave gh repo here. Why can't this refer to the local version?

Comment (tomwilkie) [inline test/units.sh]: My understanding is that go get will use the local code, and install the
binary in you bin dir.

Comment (rade) [inline test/units.sh]: > My understanding is that go get will use the local code

Yes, that is my understanding too. But it still looks weird.

Comment (tomwilkie) [inline test/units.sh]: I'll add a build target for it then.

**Pull request merged**